### PR TITLE
Dev

### PR DESCRIPTION
**Pull request by exersalza**
Added some small features
- added argparse
- readme update

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,20 +1,33 @@
...
+- [Usage](#Usage)
+- [Troubleshooting](#Troubleshooting)
+- [Known false-positives](#known-false-positives)
+- [Todo](#todo)
+- [Contact](#Contact)
+
+This is a Fivem Cipher finder for those that don't want Ciphers in their scripts :D
 
-The idea behind these scripts is to remove a Cipher or more. Currently, there is only the Python version available, but I will soon add the C++ variant.
+The idea behind these scripts is to find one or more Cipher.
```
Comment (exersalza) [inline README.md]: Maybe quick change into `hidden in your script files`

**Review by exersalza — COMMENTED** [review]
Looks good

File: README.md (hunk 1 above)
Comment (exersalza) [reply README.md]: fixed

**Comment by exersalza** [comment]
FIX THAT GOD DAMN PRINT BUG

File: cipherFinder/finder.py [hunk 2]
```diff
@@ -146,6 +151,9 @@ def check_file(d: str, file: str, count: int) -> tuple[int, int]:
 
 
 def write_log_file(**kw) -> int: 
+    if kw.pop('args').no_log:
```
Comment (exersalza) [inline cipherFinder/finder.py:154]: maybe here?

Comment (exersalza) [inline cipherFinder/finder.py:154]: maybe print bug here?

Comment (exersalza) [reply cipherFinder/finder.py:154]: Fixed

**Review by exersalza — COMMENTED** [review]
Should be fine so far

File: cipherFinder/finder.py (hunk 2 above)
Comment (ZerXGIT) [reply cipherFinder/finder.py:154]: LGTM

**Pull request merged**